### ISSUE TITLE
Comment akamai zones?

### ISSUE THREAD

**Issue by Mistraval** [ab77/netflix-proxy#198]
I suppose this is not really an issue, but it would be really nice to have comments or some sort of list of reference for all the different akamai zones and for what service they are used for. Zones like the following:

"besmenabs-s.akamaihd.net."
"pga-lh.akamaihd.net."
"fbchdvod-f.akamaihd.net."

Aren't very clear.

**Comment by ab77** [comment]
To be fair most of these were added by different people over time. Comments would be useful, so if anyone wants to Google around for obscure hosts and match them to some service, I welcome pull requests 😬 

e.g. besmenabs-s.akamaihd.net. is probably related to http://www.beinsports.com/

**Comment by ab77** [comment]
Please register for updates on the mailing list (http://netflix-proxy.belodedenko.me/) RE: August update. An alternative (paid subscription based) unblocking solution is close to being completed and should be going to trial soon.
